pypolygames/utils/multi_counter.py

```diff
@@ -8,6 +8,9 @@
 from collections import defaultdict, Counter
 from datetime import datetime
 from tensorboardX import SummaryWriter
+import wandb
+
+wandb.init(project="polygames", sync_tensorboard=True)
```

Comment:
I suggest passing in `save_code=True` to make sure git diff and current commit are saved, so the experiments are all easy to reproduce.

Also `settings=wandb.Settings(code_dir=str(Path(__file__).parent.parent.parent))` so it knows that the git root dir is where the code that should be saved lives. Though it could be that this saves all the files from `build` so you should check a run to see if that's the case.

Reply:
done